specifications/llm_specification_generator.py

```
@@ -1,104 +1,358 @@
-    def __init__(self, model: str, parsec_file: ParsecFile):
+    def __init__(
+        self,
+        model: str,
+        system_prompt: str,
+        verifier: VerificationClient,
+        verification_context_manager: VerificationContextManager,
+        parsec_file: ParsecFile,
+        num_specification_candidates: int,
+        num_repair_candidates: int,
+        num_repair_iterations: int,
+        use_cache: bool = False,
```

Ruff (0.14.10)

Boolean-typed positional argument in function definition

(FBT001)